suite-native/module-send/src/sendFormThunks.ts

```diff
@@ -221,6 +230,12 @@ export const updateSelectedFeeLevelThunk = createThunk(
         const draftCopy = { ...draft };
 
         draftCopy.selectedFee = feeLevelLabel;
+        if (feePerUnit) {
```

Comment:
those if clauses are probably not needed here
```suggestion
            draftCopy.feePerUnit = feePerUnit;
            draftCopy.feeLimit = feeLimit;
```

Reply:
Unfortunatelly it is. The assigned values has to be string, but the arguments can be undefined.
![Snímek obrazovky 2024-11-21 v 8 35 25](https://github.com/user-attachments/assets/628d8a85-17ec-4437-8f8d-9394ac1620cc)